src/_pytest/terminal.py

```diff
@@ -767,6 +778,7 @@ def getreports(self, name):
 
     def summary_warnings(self):
         if self.hasopt("w"):
+            # XXX: this needs to be fixed
```

Comment:
Not sure if you meant to keep this here or not.

Reply:
No.. it shows where the Union of the different report types causes a problem.